File: C_Sharp_Programlama/11.Nesne-Yönelimli-Programlama/11.3-Inheritance(Kalıtım)/kalıtım/Vasita.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Class_kalıtım
{
    class Vasita
    {
        private string marka, renk;
        private int model;
        private double fiyat;

        public Vasita()
        {
        }

        public void goster()
        {
            Console.WriteLine(marka+" "+model+" "+renk+" "+fiyat);
        }

        public string Marka { get => marka; set => marka = value; }
        public string Renk { get => renk; set => renk = value; }
        public int Model { get => model; set => model = value; }
        public double Fiyat { get => fiyat; set => fiyat = value; }
    }
}

File: Java_Programlama/11.0.Object-Oriented-Programming(OOP)/11.0-Class-Yapısı/vasita.java
package vasita;

/*
class Otomobil {
    String marka;
    int model;
    String renk;
    double fiyat;

    public void goster(){
        System.out.println("Marka: "+marka+
                "\nModel: "+model+
                "\nRenk: "+renk+
                "\nFiyat: "+fiyat);
    }
}
*/

public class Vasita{
    public static void main(String[] args){

        Otomobil oto1 = new Otomobil();
        oto1.marka = "Volvo";
        oto1.model = 2020;
        oto1.renk = "Gümüş";
        oto1.fiyat = 1000000;

        oto1.goster();
}